front/src/hooks/useQrResolver.ts

```diff
@@ -52,7 +57,7 @@ export const useQrResolver = () => {
         })
         .then(({ data, errors }) => {
           if ((errors?.length || 0) > 0) {
-            const errorCode = errors ? errors[0].extensions?.code : undefined;
+            const errorCode = errors ? errors[0]?.extensions?.code : undefined;
             if (errorCode === "FORBIDDEN") {
```

Comment:
the only errorCode left is if an unexpected error happens --> IQrResolverResultKind = FAIL should only be here